docs/using/administering/debugging.rst

```diff
@@ -33,6 +37,50 @@ It is possible to see the available unit names, and then view the logs with ``jo
    [root@node1]# journalctl -u flocker-container-agent
    [root@node1]# journalctl -u flocker-control
 
+Bug Reporting
+-------------
+
+When reporting issues with Flocker please include:
+
+* operating system and version,
+* Linux kernel version,
+* Docker version and configuration,
+* Flocker log files and
+* recent syslog content.
+
+The following script can be used to gather this information on Ubuntu 14.04 and CentOS 7.
+
+* :download:`flocker-log-export.py`
```

Comment:
It doesn't just export logs, so give it name that reflects that. `flocker-bug-report-dump` or something?